skopt/optimizer/optimizer.py

```diff
@@ -109,7 +109,7 @@ class Optimizer(object):
 
     """
     def __init__(self, dimensions, base_estimator,
-                 n_random_starts=10, acq_func="gp_hedge",
+                 n_random_starts=10, n_initial_points=10, acq_func="gp_hedge",
```

Comment:
Don't really like `n_starts`, it doesn't tell me "what the parameter is about". `n_initial_points` is very long to type. Maybe there is a third name we can use?